src/services/ui/src/pages/detail/index.tsx

```diff
@@ -55,7 +55,7 @@ const StatusCard = (data: OsMainSourceItem) => {
         </h2>
         {checker.hasEnabledRaiWithdraw && (
           <em className="text-xs my-4">
-            {"Withdraw Formal RAI Response - Enabled"}
+            {"Withdraw Formal RAI Response - Enabled"}{" "}
```

Comment:
We know two things to be true:

1. Substatuses can exist as multiples to one record, meaning we can have many substatuses to display at a time for a single package.

2. Substatus was important enough to become its own object property on current OneMAC.

I think we should use these two pieces of knowledge and come up with a better way to render our substatuses on the Details page. Or, at least chase down the answer to whether or not we should consider any more potential substatuses in the foreseeable future. If we are, I'd encourage you to take a look at `PackageActionsCard` below this and see if you can use its same concepts to render _n_-many substatuses without the need for adding white-space strings to separate them.